### ISSUE TITLE
[MacOs] Incorrect display if several layouts installed

### ISSUE THREAD

**Issue by herveDarritchon** [IlanCosman/tide#356]
Hi,

I have several keyboard layouts installed on my computer 

<img width="277" alt="image" src="https://user-images.githubusercontent.com/5511927/197984563-81d15d53-d7cf-48a4-b604-539267e7ca80.png">

When I setup Tide (version 5.5.1) with Fish (version 3.5.1) on my Mac (version Monterey 12.4) I have disply issues with the prompt in my terminal. Looks like japanese characters (not sure, I am just a beginner ;) ). Please look below to see how it is displayed.

<img width="727" alt="image" src="https://user-images.githubusercontent.com/5511927/197984434-44726a15-7258-4885-921e-feae49fce4dd.png">

Do you know how I can change this to have proper icons displayed ? Do you think it is a bug ?

Down below I have put my debug data from `tide debug-report` command.

Thanks

=============================================================
fish version: 3.5.1
tide version: 5.5.1
term: xterm-256color
os: MacOs Monterey 12.4
terminal emulator: iTerm 2
fish startup: 146.47 millis
fisher plugins: jorgebucaran/fisher jethrokuan/z jorgebucaran/nvm.fish andreiborisov/sponge jorgebucaran/autopair.fish IlanCosman/tide@v5

**Comment by bohrium2b** [comment]
Do you have the nerd fonts installed?

**Comment by herveDarritchon** [comment]
yes, I installed nerf fonts (https://github.com/ryanoasis/nerd-fonts#option-4-homebrew-fonts) and it is not fixed. Same issues.

**Comment by aabuhijleh** [comment]
I have the same issue

**Comment by herveDarritchon** [comment]
I suspect to issue to be linked to extra keyboard layouts. I have japanese keyboard installed on my computer with the french keyboard.

**Comment by L33t-HAXXOR** [comment]
I have only a Spanish - ISO keyboard layout installed currently
I do plan to get Japanese - Romaji working on my next install

With only the one layout, I too have this issue
